cmd/git/git.go

```diff
@@ -572,6 +573,14 @@ type DSGit struct {
 	Logger logger.Logger
 }
 
+// PublisherPushEvents - this is a fake function to test publisher locally
```

Comment:
I want to keep that function because it allows full debugging without actual connection to S3 - it fakes the connection and outputs "human-readable" data for debugging. The function is declared but never used, but when needed I can do a simple search & replace `Publisher.PushEvents` -> `PublisherPushEvents` to do full debugging.